networkit/cpp/centrality/KadabraBetweenness.cpp

```diff
@@ -190,7 +190,7 @@ void KadabraBetweenness::computeDeltaGuess() {
     const count n = G.upperNodeIdBound();
```

Comment:
If in this scope `n` is always casted to `double`, just change its type from `count` to `double`.